apps/frontend/instrumentation.ts

```diff
@@ -0,0 +1,26 @@
+import * as Sentry from '@sentry/nextjs'
+
+export const register = () => {
+  if (process.env.NEXT_RUNTIME === 'nodejs') {
```

Comment:
```suggestion
  if (['nodejs', 'edge'].includes(process.env.NEXT_RUNTIME)) {
```

속성이 동일하면 그냥 이렇게 통일하는 건 어떨까요?

Reply:
고쳤습니다~~